hack/update-crds.sh

```diff
@@ -25,4 +25,13 @@ fi
 
 operator-sdk generate k8s
 
-operator-sdk generate openapi
+operator-sdk generate crds
+
+go build -o ./bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
```

Comment:
as I mentioned in the ticket description, I think this build should be cached by the Dockerfile (and not executed here every run)

Reply:
I also don't see the contribution markdown update, which is also mentioned in the ticket description

Reply:
We'll continue with it in CSI-1296